README.md

```diff
@@ -199,7 +199,7 @@ Here is a partial list of examples:
 * Integrating with services.
   * Mozilla offers some useful services such as
     [linting](https://github.com/mozilla/addons-linter) and
-    [signing](http://olympia.readthedocs.org/en/latest/topics/api/signing.html)
+    [signing](https://github.com/mozilla/sign-addon)
```

Comment:
@axlwaii you are right, the link is outdated, the new one should be https://addons-server.readthedocs.io/en/latest/topics/api/signing.html.

would you mind to update this PR to link this one instead of the sign-addon repository? (the doc page also mentions both `web-ext sign` and `sign-addon`, as well as describing the underlying signing API).

Reply:
@rpl Thanks for reviewing the changes. I replaced the link with https://addons-server.readthedocs.io/en/latest/topics/api/signing.html as suggested.